### PR TITLE
Migrate to using Testnet enabled Core version

### PR DESCRIPTION
**Pull request by victorkirov**
# 🔘 PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

# 📜 Background
Our core version was tied to our mainnet services. The new core version allows the use of our testnet services if the wallet is set to testnet.

# 🔄 Changes
Updated all changed references to core to pass in the network where needed. Fee estimates were also updated to get them from the testnet api rather than mainnet allowing for lower fees on testnet.

Impact:
This touches all functions which called core without specifying the chain. This includes:
- All Ordinal pages
- All BRC20 pages (These won't be possible to test in testnet as we don't have a BRC-20 testnet indexer)
- All rare sats pages
- All pages with fee estimates (all non-ordinal transaction pages, but only fee estimates were touched on those)

# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@ledgerhq/hw-transport-webusb": "^6.27.13",
     "@phosphor-icons/react": "^2.0.10",
     "@react-spring/web": "^9.6.1",
-    "@secretkeylabs/xverse-core": "1.9.1",
+    "@secretkeylabs/xverse-core": "^1.9.1-4628cff",
```
Comment (teebszet) [inline package.json:10]: be careful the ^ can mess with the exact version installed

File: src/app/screens/createInscription/index.tsx [hunk 2]
```diff
@@ -482,7 +483,7 @@ function CreateInscription() {
             <ErrorModal
               key={errorCode}
               errorCode={errorCode}
-              onRetrySubmit={executeMint}
+              onRetrySubmit={executeErrorCode ? executeMint : undefined}
```
Comment (teebszet) [inline src/app/screens/createInscription/index.tsx:486]: can't retry if known error code?

**Review by teebszet — APPROVED** [review]

**Comment by DuskaT021** [comment]
@victorkirov needs conflicts resolved

File: package.json (hunk 1 above)
Comment (DuskaT021) [reply package.json:10]: yup, it will not build the local build locally properly, if the carrot is present

**Review by teebszet — DISMISSED** [review]

**Comment by DuskaT021** [comment]
@victorkirov a new conflict needs to be resolved 😊

**Review by DuskaT021 — APPROVED** [review]

**Review by teebszet — APPROVED** [review]
@DuskaT021 I changed the base branch to the release branch, so it will be merged straight into the rc

**Pull request merged**